xbmc/platform/overrides/darwin_embedded/PlatformDarwinEmbedded.h

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright (C) 2016-2020 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "platform/darwin/PlatformDarwin.h"
+
+class CPlatformDarwinEmbedded : public CPlatformDarwin
+{
+public:
+  /**\brief C'tor */
```

Comment:
doxygen doesn't provide any value here (you can remove it from all similar headers being added)